### ISSUE TITLE
blink for kmer GWAS

### ISSUE THREAD

**Issue by dosshra** [jiabowang/GAPIT#130]
I wonder if blink can be used for kmer  based GWAS  using kmer presence absence matrix without positions of the kmer on a reference genome.  
Thank you

**Comment by jiabowang** [comment]
Yes, that depends on your coding genotype for multiple mutations.
